### PR TITLE
Porting From Reflux To Mcfly

### PR DESCRIPTION
**Pull request by kenwheeler**
- Fixed overflow caused by trying int operations on string input
- Added a key in recipe details
- Replaced Reflux with McFly

### CODE REVIEW

File: client/actions.js [hunk 1]
```diff
@@ -1,17 +1,36 @@
-var Reflux = require("reflux");
-// var api = require('api')
+var McFly = require('./McFly');
```
Comment (ryan-roemer) [inline client/actions.js]: Shouldn't this be `mcFly` as an instance variable / singleton?

File: client/McFly.js [hunk 2]
```diff
@@ -0,0 +1,3 @@
+var McFly = require('mcfly');
```
Comment (ryan-roemer) [inline client/McFly.js]: Would like to see `client/mcfly.js` and not having caps in JS file names that we create.

**Comment by ryan-roemer** [comment]
@kenwheeler -- Great to see you part of the repo!!!

File: client/actions.js (hunk 1 above)
Comment (kenwheeler) [inline client/actions.js]: It should. Sorry bout that.

File: client/McFly.js (hunk 2 above)
Comment (kenwheeler) [inline client/McFly.js]: Good call

**Comment by kenwheeler** [comment]
@ryan-roemer Thanks!

**Pull request merged**